roles/apply-config-update/tasks/beta-4.yaml

```diff
@@ -0,0 +1,14 @@
+---
+- name: Test for exit_account
+  shell: grep -c "^\s{4}exit_account: lighthouse-only/exit-account.yaml" /etc/stereum/ethereum2.ya
+  register: exit_account
+  become: yes
+
+- name: Add exit-account to config
+  lineinfile:
+    dest: "/etc/stereum/ethereum2.yaml"
+    insertafter: "^\s{4}create_account: lighthouse-only/create-account.yaml"
```

Comment:
this would be better: `^\s{4}create_account:\slighthouse-only`

you can try regex here: https://regex101.com/

Reply:
okey, got it, thanks for the link